backend/push.js

```diff
@@ -219,6 +174,7 @@ export const postEvent = async (subscription: Object, event: ?string): Promise<v
   // Note: web push notifications can be 'bodyless' or they can contain a body
   // If there's no body, there isn't anything to encrypt, so we skip both the
   // encryption and the encryption headers.
+  // console.debug(`postEvent to ${subscription.id}:`, event)
```

Comment:
Stray debug?

Reply:
I think it's OK to leave here just in case someone is debugging this part of the code. I found it useful and that's why I decided to comment it out instead of removing it completely. If you really want me to remove it I can though.